src/posts/spotlights/jordan.md

```diff
@@ -10,18 +10,105 @@ mapsLink: https://www.google.com/maps/place/Jordan/@31.1675526,36.1454843,7.94z/
+  - Comfortable shoes
+  - A light backpack for carrying food and water for the day
 ---
 
-# Petra and Wadi Rum
+## Understand
 
-## Wow
+Jordan is a small country in the Middle East, often considered one of the most stable from the political and economic perspective. The country attracts more than 1 million visitors every year, most of whom are coming specially for the rock city of Petra, one of the [seven new wonders of the world](https://en.wikipedia.org/wiki/New7Wonders_of_the_World).
 
-This actually works as expected
+Further south from Petra, Jordan shares a long stretch of desert with Saudi Arabia, the land of the bedouins. Part of that area is the red desert of Wadi Rum, famous for being the filming location for a lot of Hollywood movies.
 
-Here's a _list_:
+<figure>
+  <img src="/img/jordan/wadi-rum.jpg" alt="Wadi Rum desert">
+  <figcaption>Welcome to the red desert.</figcaption>
+</figure>
 
-- one
-- two
-- three
+Petra and Wadi Rum are usually visited together due to their proximity, but of course you can choose a different itinerary.
 
-### Nice!
+## How to organize your trip
+
+We visited Jordan during [a longer trip that also covered Israel](/travel/a-taste-of-the-middle-east/). But of course you can visit Jordan separately since you can easily spend somewhere around a week touring the entire country.
+
+Since we only had a few days and getting in and out of Jordan from Israel is not that straight-forward (it's not as hard as you think though), we decided to book a trip with a Israeli company. It turned out to be the best decision as we had an amazing experience with [Abraham Tours](https://www.abrahamtours.com/) and we definitely recommend them to everyone.
+
+A lot of travel agencies in Israel offer trips to Petra and Jordan in general and if you're short on time, it is definitely your best option as they take care of everything for you.
+
+Petra and Wadi Rum are located in the south part of Jordan, so the best starting point is in Eilat, in the south of Israel, by the Red Sea. You can also book trips directly from Jerusalem or Tel Aviv, but you spend a lot of time on the road and it's not worth it. Plus, Eilat is a great destination by itself and there are plenty of things to do there, if you choose to stay a few days.
+
+So in the middle of our stay in Eilat, we booked [this 2 day trip](https://www.abrahamtours.com/tours/petra-and-wadi-rum-2-day-from-eilat/) to Jordan. Bear in mind that while the trip itself seems expensive when you consider the period, Jordan in general is a very expensive country.
+
+We were 12 people in the group, so it was a great experience overall. Someone from the travel agency organized our departure from Eilat and we were driven to the border which is just outside the city. Here, there was another person waiting on the Jordanian side (you cross the actual border by foot) that guided us to our small bus. There we met our guide, Mahmoud, who was with us for the entire duration of the trip.
```

Comment:
We were 12 people in the group, and it was a great experience overall. Someone from the travel agency organized our departure from Eilat and we were driven to the border, which is just outside the city. We crossed the actual border on foot, and there was another person waiting for us on the Jordanian side. He helped us with the legal formalities and guided us to our small bus. There we met our guide, Mahmoud, who was with us for the entire duration of the trip.